apps/web/hooks/blockchain/use-graduation-nft.ts

```
@@ -0,0 +1,388 @@
+ *     <button onClick={handleMintNFT} disabled={isLoading}>
+ *       {isLoading ? 'Minting...' : 'Mint Graduation NFT'}
+ *     </button>
+ *   )
+ * }
+ * ```
+ */
+export function useGraduationNFT() {
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+  const [nftCache, setNftCache] = useState<Map<Address, GraduationNFT | null>>(new Map())
+  const [eligibilityCache, setEligibilityCache] = useState<Map<Address, any>>(new Map())
```

Biome (2.1.2)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)